tests/dist_test.py

```diff
@@ -0,0 +1,24 @@
+import unittest
```

Comment:
Add all the tests in one class as three functions. And a third message parameter for each test. This is the message that will be displayed if the test fails

Reply:
I also believe each test needs to start with a "test_"

Reply:
I don't believe condensing the classes into one function is the proper way to test. For testing the stats class I need to utilize setUp and tearDown. As Stats doesn't use have any methods its simply initialized repeatedly as needed. 

As example from [25.3.4](https://docs.python.org/2/library/unittest.html#organizing-test-code)
`import unittest

class SimpleWidgetTestCase(unittest.TestCase):
    def setUp(self):
        self.widget = Widget('The widget')

class DefaultWidgetSizeTestCase(SimpleWidgetTestCase):
    def runTest(self):
        self.assertEqual(self.widget.size(), (50,50),
                         'incorrect default size')

class WidgetResizeTestCase(SimpleWidgetTestCase):
    def runTest(self):
        self.widget.resize(100,150)
        self.assertEqual(self.widget.size(), (100,150),
                         'wrong size after resize')`

Each test is organized into its own test class. which is divided as its own class that takes the set up module as an input. This allows me to run tests repeatedly on  while setting up and tearing down the class for each test

Reply:
I'm not sure that this kind of structure of tests is working under nosetests runner since it's what we use to run tests, not unittest.main() direct call. Besides, I think having a structure of subclasses is quite an overkill for just a single distribution and overcomplicates the overall design of tests. You can just define each test case with "test_" substring. A good example is how boto3 (Python API for AWS) is tested:
https://github.com/boto/boto3/blob/develop/tests/unit/test_boto3.py

Reply:
Use "import dist_function.stats" instead of "import stats". This is the fix I requested.

Also, I don't see any instance of 'stats' class in the test. And it seems that the class is not static.